mycology.b98

```diff
@@ -234,9 +234,9 @@ n                >na"stcelfer G :DAB";   >0a"detcelfer TTM :DAB"a
 > ay     v     >#^G0" sa ytiralunarg kcolc sevig G :FEDNU">:#,_$.0a"sdnocesorcim">:#,_$#vT0a"M erofeb dellac nehw detcelfer evah dluohs T :DAB"^$
  ^"HRTI"0<             v_v#!p00:-1<*ff       .$_,#! #:<"UNDEF: T after M pushed "0T^#  ;>  a"M erofeb dellac nehw detcelfer T :DOOG">:#,_#^S0" dehsup S :FEDNU">:#,_$.a,#^M#;<
 v                      >;>yynyn00g^;  #vT 0" dehsup ,sy 576 retfa ,T dnoces a dna">:#,_$.a,#vE #vT0a"tcelfer t'nseod TE :DAB"                   ^
-v )g22g33,a <                         ;>n^;                  $_,#! #:<"GOOD: ET reflected"a0># #<0a"stcelfer E :DAB"                            ^
-                                                                 v
-                                                               H >0a"tsaehtron seog v< >H retfa ^ :DAB"v             vv  $_,#! #:<H"GOOD: |_ seem to work in hovermode"av
+v )g22g33,a <                  < v"di";>n^;"d not reflect"Ev#$_,#! #:<"GOOD: ET reflected"a0># #<0a"stcelfer E :DAB"                            ^
+         >;#"UNDEF: E without M "<;>:#,_$v     ;"reflected"<     v

```

Comment:
The strings should probably be null-terminated in case `E` actually pushes something non-zero. (Ideally I suppose that would be detected too, but this is fine.)

There should also be an `a` pushed to get the blank line separator between fingerprints.

Reply:
Added the newline and null terminators to the strings in the second commit.
As for `E` pushing something non-zero, i felt safe to assume that `E` not touching the stack at all would count as "at least somewhat sane" behaviour, given how much different that would be from its behaviour when used normally.

Reply:
Oh, right, I confused `E` and `T`. Makes sense!